project.json

```diff
@@ -13,8 +14,11 @@
 		"Microsoft.AspNetCore.Server.Kestrel": "1.0.1",
 		"Microsoft.AspNetCore.StaticFiles": "1.0.0",
 		"Microsoft.Extensions.Configuration.EnvironmentVariables": "1.0.0",
-		"Microsoft.Extensions.Logging.Console": "1.0.0"
-	},
+    "Microsoft.Extensions.Logging.Console": "1.0.0",
+    "Microsoft.AspNetCore.Mvc": "1.0.1",
+    "Microsoft.AspNetCore.Routing": "1.0.1",
+    "Microsoft.AspNetCore.Razor.Tools": "1.0.0-preview2-final",
+    "Microsoft.Extensions.Configuration.Json": "1.0.0"	},
```

Comment:
`Microsoft.Extensions.Configuration.Json` should be removed as it isn't being used.